deploy/ecs/gprofiler_task_definition.json

```diff
@@ -6,9 +6,7 @@
         {
             "name": "granulate-gprofiler",
             "image": "index.docker.io/granulate/gprofiler:latest",
-            "memory": "1024",
```

Comment:
Is that the case for memory as well? I was under the impression that `memory` is respected under the container, but `cpu` is not.

Reply:
You're right I'll keep it in container level.